layout/_partials/sidebar/site-overview.njk

```diff
@@ -27,12 +24,9 @@
       {%- set categoriesPageQuery = site.pages.find({type: 'categories'}, {lean: true}) %}
       {%- set hasCategoriesPage = categoriesPageQuery.length > 0 %}
       <div class="site-state-item site-state-categories">
+        {%- set category_dir = theme.menu.categories.split('||')[0] | trim if theme.menu.categories else config.category_dir + '/' %}
         {%- if hasCategoriesPage %}
-          {%- if theme.menu.categories %}
-            <a href="{{ url_for(theme.menu.categories.split('||')[0] | trim) }}">
-          {% else %}
-            <a href="{{ url_for(config.category_dir) + '/' }}">
-          {%- endif %}
+          <a href="{{ url_for(category_dir) }}">
```

Comment:
More as a future refactor suggestion. The `<a>` anchor may persist (without the above `if` code) and has no `href` attribue (when the `if` fails), to represents a placeholder as:

> If the a element has no href attribute, then the element represents a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element's contents.
> —— [The a element | HTML Standard](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)

While it breaks current stylesheets. It required us to use `a[href]` to identify the valid and invalid ones to set the `color`.